### PR TITLE
Add background image to login screen + ability to press enter to log in

### PR DESCRIPTION
**Pull request by tarasIV**

### CODE REVIEW

File: src/app/login/login/login.component.scss [hunk 1]
```diff
@@ -1,6 +1,17 @@
-body {
+.picture {
+	background-image:url('../../../assets/login.jpg');
```
Comment (dmitryy90) [inline src/app/login/login/login.component.scss]: convert it to base64

Comment (tarasIV) [reply src/app/login/login/login.component.scss]: Converted

File: src/app/admin-panel/admin-panel/groups/groups.component.scss [hunk 2]
```diff
@@ -3,25 +3,42 @@
...
     height:40px ;
 		font-size:15px;
-}
+	}
 
-div {
-    text-align:center;
+::ng-deep body .ui-button.ui-state-default.ui-button-success  {
+	&:enabled:active, &:enabled:hover{
+	background-color: blue;
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/groups/groups.component.scss]: indentation

File: src/app/admin-panel/admin-panel/groups/groups.component.scss [hunk 3]
```diff
@@ -3,25 +3,42 @@
...
+	}
+	background-color: blue;
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/groups/groups.component.scss]: add empty line before

File: src/app/admin-panel/admin-panel/groups/groups.component.scss [hunk 4]
```diff
@@ -3,25 +3,42 @@
...
 }
 
 .groups-container {
     width:100%;
-    margin:0 auto;
+		margin:0 auto;
+
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/groups/groups.component.scss]: remove empty line

**Pull request merged**